src/lib.rs

```diff
@@ -70,19 +70,19 @@ extern "C" {
 pub fn height_to_flags(height: u32) -> u32 {
 
     let mut flag = VERIFY_NONE;
-    if height > 170059 {
+    if height >= 173805 {
```

Comment:
I went searching for _why_ this value, for anyone else who doesn't know, this was the block that P2SH was activated (by BIP16).